package.json

```diff
@@ -39,13 +39,14 @@
     "karma-jasmine": "^1.0.2",
     "karma-junit-reporter": "^1.2.0",
     "karma-spec-reporter": "0.0.26",
-		"ng-infinite-scroll": "1.3.0",
+    "ng-infinite-scroll": "1.3.0",
     "run-sequence": "^2.2.1"
   },
   "dependencies": {
-    "npm": "^5.6.0",
+    "buffer-to-vinyl": "^1.1.0",
```

Comment:
can you provide some info about where are we using this?

Reply:
[It was already in dev]( https://github.com/Videona/platform-frontend/blob/dev/package.json#L49), added in 1096efe ([see line](https://github.com/Videona/platform-frontend/commit/1096efe61977cc6dffeedc2aa23c164c98b3d708#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R47)). 

Anyway, it's being used for gulp to [pipe angular app configs in a file](https://github.com/Videona/platform-frontend/commit/1096efe61977cc6dffeedc2aa23c164c98b3d708#diff-b9e12334e9eafd8341a6107dd98510c9R34) ;)